services/IoT/applications/libraries/python.md

```diff
@@ -266,7 +262,7 @@ By default, when you subscribe to device status, status updates are received for
 ## Handling status updates from devices
 {: #handling_device_updates}
 
-To process the status updates received by your subscriptions you need to register an event callback method. The messages are returned as an instance of the Status class.
+To process the status updates that are received by your subscriptions, you need to register an event callback method. The messages are returned as an instance of the Status class.
 
 The following properties are set for both "Connect" and "Disconnect" status events:
```

Comment:
Line 278: "The following properties are only set when the action is "Disconnect":" - but line 267 says  that the propreties are set for "both "Connect" and "Disconnect" status events" - so what are these, actions or status events? need to standardize terminology and use consistently.